sections/collapsible-content.liquid

```diff
@@ -0,0 +1,382 @@
+{{ 'component-accordion.css' | asset_url | stylesheet_tag }}
+{{ 'collapsible-content.css' | asset_url | stylesheet_tag }}
+
+<div class="collapsible-content{% if section.settings.layout == 'no_container' %} color color-{{ section.settings.color_scheme }}{% endif %}{% unless section.settings.layout == 'no_container' %} color-{{ section.settings.container_color_scheme }}{% endunless %}{% if section.settings.layout == 'section_container' %} page-width{% endif %}">
+  <div class="collapsible-content-wrapper{% if section.settings.layout == 'section_container' %} color color-{{ section.settings.color_scheme }}{% endif %}">
+    <div class="{% if section.settings.image == blank %} collapsible-content-wrapper--narrow{% else %} page-width{% endif%}">
+      <div style="text-align: {{ section.settings.heading_alignment }}">
+        {%- if section.settings.caption != blank -%}
+          <p class="caption-with-letter-spacing"> {{ section.settings.caption }}</p>
+        {% endif %}
+        {%- if section.settings.heading != blank -%}
+          <h2 class="h1">{{ section.settings.heading }}</h2>
+        {% endif %}
+      </div>
+      <div class="grid grid--1-col grid--2-col-tablet collapsible-content__grid{% if section.settings.desktop_layout == 'image_first' %} collapsible-content__grid--reverse{% endif %}">
+        <div class="grid__item grid__item--content">
+          {%- for block in section.blocks -%}
+            <div class="accordion{% if section.settings.layout == 'row_container' %} color color-{{ section.settings.color_scheme }}{% endif %}" {{ block.shopify_attributes }}>
+              <details id="Details-{{ block.id }}-{{ section.id }}"{% if section.settings.open_first_collapsible_row and forloop.first %} open{% endif %}>
+                <summary>
+                  <div class="summary__title">
```

Comment:
This generates invalid HTML. Now that I inspect and see what it does, I'm not sure if there's any way around this. Even changing to `span` this leads to another invalid issue of a heading not being a valid child of `span`. 🤔